ortools/constraint_solver/samples/cvrp_reload.py

```diff
@@ -121,12 +121,15 @@ def manhattan_distance(position_1, position_2):
 def create_distance_evaluator(data):
     """Creates callback to return distance between points."""
     _distances = {}
+    BIG_DISTANCE=10000
     # precompute distance between location to have distance callback in O(1)
     for from_node in xrange(data['num_locations']):
         _distances[from_node] = {}
         for to_node in xrange(data['num_locations']):
             if from_node == to_node:
                 _distances[from_node][to_node] = 0
+            elif from_node in [0, 1, 2, 3, 4, 5] and to_node in [0, 1, 2, 3, 4, 5]:
```

Comment:
you can use `range(6)`

Reply:
yep.  

This was an after thought, then I tossed in 0 too.  Lazy programming

I also noticed xrange everywhere.  Is that old python?  I've only ever used range.

Reply:
when we keep supporting python2.7, (until v7.8, and v8.0 unofficially) we used to use `six` and `xrange()` instead of python3 range since it was faster i.e. python2 xrange IS python3 range using generator instead of building list.
Now we just have drop python2.7 compatibility, try to remove all six import so you should use `range(6)` and say good bye to python2.7